products/sle/main.pm

```diff
@@ -640,6 +640,7 @@ sub load_virt_feature_tests {
         loadtest 'virtualization/universal/hotplugging_cleanup';
     }
     loadtest "virtualization/universal/storage" if get_var("ENABLE_STORAGE");
+    loadtest "virt_autotest/libvirt_extend_storage_lvm" if get_var("ENABLE_STORAGE");
```

Comment:
The two storage test modules would be better if they are called in one if condition.

Reply:
yes, had updated now.